refactor(transformer): rename methods

overlookmotel opened the pull request:
Pure refactor. Rename a few methods when had `file_name` in their name. Convention throughout codebase appears to be `filename` (without the `_`).

overlookmotel commented:
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/6190?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#6190** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/6190?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈
* **#6189** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/6189?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#6188** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/6188?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#6186** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/6186?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#6185** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/6185?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#6184** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/6184?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#6183** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/6183?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @overlookmotel and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="11px" height="11px"/> <b>Graphite</b></a>
  <!-- Current dependencies on/for this PR: -->

Boshen commented:
### Merge activity

* **Oct 1, 3:59 AM EDT**: [Graphite](https://app.graphite.dev/github/pr/oxc-project/oxc/6190) rebased this pull request after merging its parent, because this pull request is set to merge when ready.

overlookmotel committed:
refactor(transformer): rename methods (#6190)

Pure refactor. Rename a few methods when had `file_name` in their name. Convention throughout codebase appears to be `filename` (without the `_`).
